src/pythonic/index.md

```diff
@@ -174,6 +174,9 @@ class Sieve:
         primal.sieve_destroy(self.sieve)
 
     def is_prime(self, n):
+        if n > self.upper_bound():
+            raise IndexError("{} not in upper bound {}"
```

Comment:
Looks good to me. It may just be markdown doing funny things to your formatting, but you'll need to make sure it's all contained within a single doc-comment otherwise it won't compile.

i.e.

```rust
/// Checks whether a number is prime. A non-zero response indicates `true`.
///
/// # Panics 
///
/// If `n` is greater than `upper_bounds`.
#[no_mangle]
pub unsafe extern "C" fn sieve_is_prime(sieve: *const Sieve, n: libc::c_uint) -> 
  libc::int8_t {
      (&*sieve).is_prime(n as usize) as libc::int8_t
  }
```